.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -8,15 +8,26 @@ body:
   - type: markdown
     attributes:
       value: |
-        Thank you for taking your time to report this bug!
-        Make sure you are running the latest version before reporting.
-  - type: input
+        Thank you for taking your time to report this bug! You gotta make sure these before reporting...
+        - You are running the latest version of Connector and its dependencies.
+        - This bug wasn't already reported or being closed.
```

Wording can be improved here
```suggestion
        - This bug wasn't already reported or closed as not planned.
```